### ISSUE TITLE
Support for wifi-only devices (f. e. tablets)

### ISSUE THREAD

**Issue by NeXXor** [signalapp/Signal-Android#1153]
It would be a nice feature, if TextSecure would run wifi-only devices like tablets.

**Comment by bobobo1618** [comment]
Sorry to revive a dead thread but I searched and couldn't find any updates.

@moxie0 are there any updates on multi-device support? Is there an issue I couldn't find tracking the feature? Is there any kind of public area where we can follow progress?

**Comment by ChaoticMind** [comment]
@bobobo1618 You can watch the development of the [browser extension](https://github.com/WhisperSystems/TextSecure-Browser). I would assume that Its release is a strong indication that a version of multi-device support is ready at least in some form.

**Comment by flamsmark** [comment]
Why is multi-device support being conflated with wifi-only device support?

The version of TextSecure in the Play Store is still not marked as compatible with wifi-only devices. Since the Play Store is the recommended way to install TextSecure, I'd suggest that this issue is not actually resolved — people with wifi-only devices *can't* install TextSecure the recommend way.

**Comment by agerwick** [comment]
Why is this issue closed? I can't see it having been resolved. I just installed Signal on an Android phone without a sim card, using the verification by phone (to my dumb-phone, where I keep my sim card), but in Google Play store any device in our household that are designed to not have a sim card (i.e. tablets) are considered as incompatible. 

I don't see why:
1. using a phone number as an ID can be the cause of this issue, as I might as well use a tablet and receive a verification call on another phone, just like I did on my sim-less android phone.
2. there is a need to use phone number as the only form of ID, why not use email addresses, or a used generated string for that mater?
3. the UI not being designed for tablets can be a significant problem. I've used lots of other apps that were "designed for phones", and they work just fine. Not optimal perhaps but better than nothing!
